agent/config/types.go

```diff
@@ -157,6 +157,9 @@ type SensitiveRawMessage struct {
 // NewSensitiveRawMessage returns a new encapsulated json.RawMessage that
 // cannot be accidentally logged via .String/.GoString/%v/%#v
 func NewSensitiveRawMessage(data json.RawMessage) *SensitiveRawMessage {
```

Comment:
Please update the documentation that this can now return `nil` when invoked with empty `data`